### DISCUSSION TITLE
Illigal user App!

### DISCUSSION THREAD

**Discussion by Arfaniel** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2777] [Q&A]
Have this error, and it never goes away!
I have btt tft35 with GD32F207 chip.  Using BIGTREE_GD_TFT35_V3.0.27.x.bin (latest for April 19, 2023), config.ini from this repo, and a folder with icons....
Steps to reproduce:
1. Flashed my screen to new firmware via SDcard - successfully.
2. Check all options - successfully
3. TestPrint from SDcard - successfully 
4. Turn off my printer
5. Turn on printer with SDcard inserted (no firmware there, just some gcode-files)

Result: 
![0-02-05-bf5cc958ea87183caf57b611dc0d2dcfba4093ab941c213e1a03db83f1fdfc3a_1c6db1d62609f5](https://user-images.githubusercontent.com/13827433/233106250-31da30e3-1031-4f85-9a14-e9db41e52b3d.jpg)

total crush of firmware... have to flash every time when I forget to remove SDcard before turning on printer!!!

**Comment by burneme** [comment]
Hi Arfaniel,
could it be possible that a hidden file with the extension ".bin" is on your card? Somehow it looks like the system finds that sort of file and tries to install it. I recommend to fully reformat that sd-card to be sure to get rid of all rubbish. 
Good luck.
